subsys/logging/log_backend_uart.c

```diff
@@ -136,6 +136,13 @@ static void log_backend_uart_init(struct log_backend const *const backend)
 	}
 }
 
+static int is_ready(struct log_backend const *const backend)
+{
+	if (backend->cb)
```

Comment:
curly braces here

Reply:
Updated. But it seems there is another CI fail